scraper.py

```diff
@@ -5,31 +5,39 @@
 import csv
 import sys
 import mysql.connector
+from plyer import notification
 
-c = sys.argv[1]
-yr = sys.argv[2]
+c = sys.argv[0]
+yr = sys.argv[0]
 if (c == "ec"):
     st = "PES2"
 else:
     st = "PES1"
 
 
+notification.notify(
+    title = "redirect",
+    message = 'You are being redirected to the telegram bot',
+    timeout = 5
+)
+
```

Comment:
I think I clarified this with you and you've made significant progress over it too. Which maintainer told you that the meaning of this issue was to send a desktop notification?